plugins/node/opentelemetry-instrumentation-mongodb/README.md

```diff
@@ -20,7 +20,7 @@ npm install --save @opentelemetry/instrumentation-mongodb
 
 ### Supported Versions
 
-- `>=3.3 <4`
+- `>=3.3`
```

Comment:
nit: this should be `>=3.3 <5`. If mongo releases version 5, it will not be patched

Reply:
Fixed